pkg/push/push.pb.go

```diff
@@ -0,0 +1,1292 @@
+		return this == nil
+	}
+
+	that1, ok := that.(*EntryAdapter)
+	if !ok {
+		that2, ok := that.(EntryAdapter)
+		if ok {
+			that1 = &that2
+		} else {
+			return false
+		}
+	}
+	if that1 == nil {
+		return this == nil
+	} else if this == nil {
+		return false
+	}
+	if !this.Timestamp.Equal(that1.Timestamp) {
+		return false
+	}
+	if this.Line != that1.Line {
+		return false
+	}
+	return true
+}
+func (this *PushRequest) GoString() string {
+	if this == nil {
+		return "nil"
+	}
+	s := make([]string, 0, 5)
+	s = append(s, "&push.PushRequest{")
+	s = append(s, "Streams: "+fmt.Sprintf("%#v", this.Streams)+",\n")
+	s = append(s, "}")
+	return strings.Join(s, "")
+}
+func (this *PushResponse) GoString() string {
+	if this == nil {
+		return "nil"
+	}
+	s := make([]string, 0, 4)
+	s = append(s, "&push.PushResponse{")
+	s = append(s, "}")
+	return strings.Join(s, "")
+}
+func (this *StreamAdapter) GoString() string {
+	if this == nil {
+		return "nil"
+	}
+	s := make([]string, 0, 7)
+	s = append(s, "&push.StreamAdapter{")
+	s = append(s, "Labels: "+fmt.Sprintf("%#v", this.Labels)+",\n")
+	if this.Entries != nil {
+		vs := make([]*EntryAdapter, len(this.Entries))
+		for i := range vs {
+			vs[i] = &this.Entries[i]
+		}
+		s = append(s, "Entries: "+fmt.Sprintf("%#v", vs)+",\n")
+	}
+	s = append(s, "Hash: "+fmt.Sprintf("%#v", this.Hash)+",\n")
+	s = append(s, "}")
+	return strings.Join(s, "")
+}
+func (this *EntryAdapter) GoString() string {
+	if this == nil {
+		return "nil"
+	}
+	s := make([]string, 0, 6)
+	s = append(s, "&push.EntryAdapter{")
+	s = append(s, "Timestamp: "+fmt.Sprintf("%#v", this.Timestamp)+",\n")
+	s = append(s, "Line: "+fmt.Sprintf("%#v", this.Line)+",\n")
+	s = append(s, "}")
+	return strings.Join(s, "")
+}
+func valueToGoStringPush(v interface{}, typ string) string {
+	rv := reflect.ValueOf(v)
+	if rv.IsNil() {
+		return "nil"
+	}
+	pv := reflect.Indirect(rv).Interface()
+	return fmt.Sprintf("func(v %v) *%v { return &v } ( %#v )", typ, typ, pv)
+}
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ context.Context
+var _ grpc.ClientConn
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the grpc package it is being compiled against.
+const _ = grpc.SupportPackageIsVersion4
+
+// PusherClient is the client API for Pusher service.
+//
+// For semantics around ctx use and closing/ending streaming RPCs, please refer to https://godoc.org/google.golang.org/grpc#ClientConn.NewStream.
+type PusherClient interface {
+	Push(ctx context.Context, in *PushRequest, opts ...grpc.CallOption) (*PushResponse, error)
+}
+
+type pusherClient struct {
+	cc *grpc.ClientConn
+}
+
+func NewPusherClient(cc *grpc.ClientConn) PusherClient {
+	return &pusherClient{cc}
+}
+
+func (c *pusherClient) Push(ctx context.Context, in *PushRequest, opts ...grpc.CallOption) (*PushResponse, error) {
+	out := new(PushResponse)
+	err := c.cc.Invoke(ctx, "/push.Pusher/Push", in, out, opts...)
```

Comment:
I'm worried this may be a breaking change since the path is used in the method name here.

Reply:
can I write some test to check if I broke something? What does this path affect?